File: 3. Conditional Loops/1. Cpp/1. Basic/areaofequilateraltriangle.cpp
#include <bits/stdc++.h>
using namespace std;
#define ll long long
#define FIO ios_base::sync_with_stdio(false), cin.tie(0), cout.tie(0)
#define pi 3.14
#define endl "\n"

int main() {
	FIO;
	freopen("input.txt", "r", stdin);
	freopen("output.txt", "w", stdout);

	double side;
	cin >> side;
	double part1 = side * side;
	double part2 = (sqrt(3)) / 4;
	double area = part1 * part2;
	if (side <= 0) {
		cout << "Side cannot be negative" << endl;
	}
	else {
		cout << "Area is: " << area << endl;
	}
	return 0;
}

File: 3. Conditional Loops/2. Java/1. Basic/areaofequilateraltriangle.java
import java.util.Scanner;
import java.math.*;
public class areaofequilateraltriangle {
    private static final double pi = 3.1416;
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter the side: ");
        double side = sc.nextDouble();
        double part1 = (Math.sqrt(3))/4;
        double part2 = (side*side);
        double area = part1*part2;
        if(side<0){
            System.out.println("Area is negative");
        }
        else{
            System.out.println("Area is: " + area);
        }
        sc.close();
    }
}
